### DISCUSSION TITLE
neuromodulation (tDCS/TMS)  MRI public datasets Or links

### DISCUSSION THREAD

**Discussion by xiongyy99255** [simnibs/simnibs#265] [2. General]
Dear SimNIBS team and experts, thanks for the great tool!
      I would like to ask you if there are any MRI public datasets related to neuromodulation (tDCS/TMS). Or links that might be able to be accessed!
Thanks!

**Comment by Kstoffer** [comment]
If what you are searching for is a dataset where you can perform simulations there are many options for example the HCP dataset (https://www.humanconnectome.org/) or OpenNeuro.
